apps/framework-cli/src/cli.rs

```diff
@@ -156,7 +191,17 @@ async fn top_command_handler(
             // TODO: refactor this to be extracted in different functions
             match template {
                 Some(template) => {
-                    templates::generate_template(template, CLI_VERSION, dir_path).await
+                    let _ = templates::generate_template(template, CLI_VERSION, dir_path).await;
```

Comment:
do not ignore the `Result` type. Use `?`
https://doc.rust-lang.org/rust-by-example/std/result/question_mark.html

Reply:
how would `?` be used here? _rustc_ complains when i use `?` and suggests the `_` ignore syntax instead.

<img width="1428" alt="Screenshot 2024-07-11 at 12 07 50 PM" src="https://github.com/514-labs/moose/assets/958603/f94e6fad-13a6-42e5-9bb6-f5e25b3922ff">

Reply:
Right. There's a `RoutineSuccess` with the message `Run these commands to start Moose: cd {}/moose && npx @514labs/moose-cli@latest dev`.

If we don't want that message any more, change the return type of `generate_template` to `Result<(), RoutineFailure>`.